org.eclipse.lsp4j.jsonrpc/src/main/java/org/eclipse/lsp4j/jsonrpc/messages/ResponseError.java

```diff
@@ -16,16 +16,20 @@
 	 * A number indicating the error type that occured.
 	 */
 	@NonNull
-	private ResponseErrorCode code;
+	private int code;
 
-	public ResponseErrorCode getCode() {
+	public int getCode() {
 		return this.code;
 	}
 
-	public void setCode(ResponseErrorCode code) {
+	public void setCode(int code) {
```

Comment:
@svenefftinge The protocol says that for requests custom error codes can be defined and the version 3.0 there is such for the initialization request.